src/sys/uio.rs

```diff
@@ -177,59 +187,3 @@ pub fn process_vm_readv(
     Errno::result(res).map(|r| r as usize)
 }
 }
-
-/// A vector of buffers.
-///
-/// Vectored I/O methods like [`writev`] and [`readv`] use this structure for
-/// both reading and writing.  Each `IoVec` specifies the base address and
-/// length of an area in memory.
-#[repr(transparent)]
-#[derive(Clone, Copy, Debug, Eq, Hash, PartialEq)]
-pub struct IoVec<T>(pub(crate) libc::iovec, PhantomData<T>);
```

Comment:
It might help users update their codebases if you leave this struct definition in place, but mark it and all of its methods as `#[deprecated]`.